### ISSUE TITLE
1.3.x - phpcs: Unable to locate phpcs. Please add phpcs to your global path or use composer depency manager to install it in your project locally.

### ISSUE THREAD

**Issue by zmni** [ikappas/vscode-phpcs#15]
Works fine on version 1.2.x, but shortly after update to above version, vscode no longer detect/locate phpcs. Global path configure properly, on default setting, hardcode setting to absolute path, I try everything, nothing works.

Output from Developer Tools:

```
phpcs: Unable to locate phpcs. Please add phpcs to your global path or use composer depency manager to install it in your project locally.
    e.doShow                  @ messageService.ts:119
    e.show                    @ messageService.ts:103
    (anonymous function)      @ mainThreadMessageService.ts:53
    i.Class.derive._oncancel  @ winjs.base.raw.js:1641
    e.$showMessage            @ mainThreadMessageService.ts:23
    e.handle                  @ abstractThreadService.ts:34
    s                         @ ipcRemoteCom.ts:269
    f                         @ ipcRemoteCom.ts:226
    _combinedTickCallback     @ internal/process/next_tick.js:67
    _tickCallback             @ internal/process/next_tick.js:98
```

**Comment by ikappas** [comment]
@zmni  Can you please check whether this issue is resolved by 0.7.0?

**Comment by ikappas** [comment]
I assume that this issue has been resolved in 0.7.0 so closing. Feel free to open a new issue if you do have problems
